### DISCUSSION TITLE
Increase wire_holograms_max

### DISCUSSION THREAD

**Discussion by MorlandMoran** [wiremod/wire#2438] [Suggestions]
The holo limit hasn't changed since around 2014, but as servers have become much more powerful, it now often happens that prop limits are greater than hologram limits, despite the fact that holograms are infinitely less jerky than genuine props with collision.

To enable for better creation that would operate on every server as the default setting and not just on select servers that manually upped it, I'd advise 300-350 as a great new limit.

**Comment by MorlandMoran** [comment]
Additionally, the update time is only a matter of seconds, and many people have complained that the current limit is too low to make advanced creations.

**Reply by thegrb93** [reply]
That's up for the server owner to decide. I think the default limit matches the default prop limit.

**Reply by MorlandMoran** [reply]
> 

No it doesn't, prop limits are now in sandbox above 500 easily, even on DarkRP it is at around 270, so it makes no sense really, it should have been increased during years to match with current servers performances that can handle much more than it used to do in 2014, it's already been increased on the request of shadowscion there is no reason to not increase it again 8 years later.

**Reply by deltamolfar** [reply]
> That's up for the server owner to decide. I think the default limit matches the default prop limit.

In my opinion, holo limit should not correspond to the prop limit. First of all, it's easy to just use multiple E2. And by default, the limit of E2 per user is around 10-20. + Dont forget that holos has no physics or collisions, so they can't provoke crazy physics. + By default, modelany == 0, so most of the time, it's not render heavy. I'm leadership/dev of one of the biggest ukranian sandbox servers atm (~20 stable online), and using wire/E2 is the current meta (Anti Air, radars, drones, etc.). Never did E2 cause us lags, even tho our limit is at 350, we have modelany == 1, and scale limit is two times bigger than default. Of course, you can cause lags with holo and a lot of players if you want to - but so you can do with a lot of other features. E2 is already powerful enough to make havoc on bad moderated servers. So I don't understand why to limit holos here, while you can spawn 4 props a second, control shit ton of props a second, etc, etc, and cause problems up to crash with current limitations.

**Comment by Fasteroid** [comment]
I concur that we need more—my mech I made with my holo structure lib is already at 275.  There's no way I can easily shave 25 holos to drop within the limit.  An extra 50 (to bring the limit from 250 to 300) would probably go a long way, and doesn't feel too unreasonable.

**Reply by InfHorus** [reply]
Send pic of the mech

**Reply by Fasteroid** [reply]
I'll do you one better, here's a video:
https://www.youtube.com/watch?v=FBywzYRR59c

**Reply by Fasteroid** [reply]
Also, I did manage to shave 25—I removed some detail on the gun barrels and halved the number of holos in the fireball structure.  It doesn't look as good, but will have to do for now.

**Reply by InfHorus** [reply]
Looks fun

**Comment by Hazeofdream** [comment]
> That's up for the server owner to decide. I think the default limit matches the default prop limit.

server owners also unironically think that 400 ops e2s are the cause of their servers murder, and that their is absolutely no reason why anyone should enable any e2 core (its too abusive)

**Reply by Fasteroid** [reply]
Perhaps this is evidence not that we shouldn't increase the default hologram limit, but instead that OPS is a fundamentally flawed and misunderstood system.  Pretty sure @Vurv78 will back me up on this.

**Reply by vurvdev** [reply]
It's a good system, just people were too lazy to assign proper values for the majority of functions
